nexus-repository-apk-it/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>apk-parent</artifactId>
-    <version>0.0.1</version>
+    <version>0.0.0-SNAPSHOT</version>
```

Comment:
in the readme, we have just 0.0.1 here it is -with Snapshot. Need to align to one stile

Reply:
Thank you! I'll reach out to @bhamail and @DarthHater for clarification, I was under the impression circleCI was going to rev it to 0.0.1 but i have a feeling my understanding was wrong.